From 4f15b08033441eeb2004c2a7ec19c91e1e75c332 Mon Sep 17 00:00:00 2001
From: The-going <48602507+The-going@users.noreply.github.com>
Date: Mon, 22 Nov 2021 18:53:11 +0300
Subject: [PATCH] wifi 4002 realtek 8723cs Fix IEEE80211_MAX_AMPDU_BUF
 redefined

After v4.19 linux kernel, this definition is not required.
The 1 drivers/net/wireless/realtek/rtlwifi/base.c file uses
the variable IEEE80211_MAX_AMPDU_BUF_HT.

See `git log -p b8042b3da925f390c1482b -3` command in
git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git
---
 drivers/net/wireless/realtek/rtl8723cs/include/wifi.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/net/wireless/realtek/rtl8723cs/include/wifi.h b/drivers/net/wireless/realtek/rtl8723cs/include/wifi.h
index 9d21a5afd..8ef0fd7ff 100644
--- a/drivers/net/wireless/realtek/rtl8723cs/include/wifi.h
+++ b/drivers/net/wireless/realtek/rtl8723cs/include/wifi.h
@@ -1006,7 +1006,6 @@ typedef enum _HT_CAP_AMPDU_DENSITY {
  * According to IEEE802.11n spec size varies from 8K to 64K (in powers of 2)
  */
 #define IEEE80211_MIN_AMPDU_BUF 0x8
-#define IEEE80211_MAX_AMPDU_BUF 0x40
 
 
 /* Spatial Multiplexing Power Save Modes */
-- 
2.33.1

